### DISCUSSION TITLE
temperature at startup of VWL 75/6

### DISCUSSION THREAD

**Discussion by avanmourik** [john30/ebusd#1359] [Q&A] [answered]
When my heatpump is started, the ZWZ AI indicates a desired value for the water temperature to the heating circuit. So far I have not been able to find the required line in the ctlv2 configuration file to log this setpoint. Once the pump is running the setpoint is gradually lowered (modulation) to match the setpoint given by the heating curve.
Doe anyone know which variable I should use to monitor this

**Comment by ralfnap2** [comment]
Hi,
ebusctl read  SetMode
returns
auto;39.5;-;-;0;1;1;0;0;0
39.5 is target value
****
ebusctl read  Status01
39;34.5;-;-;-;on

current temp 39
return flow 34.5

**Reply by avanmourik** [reply]
Dear ralfnap2,
Thanks for your reply
Great! There is a way to get to the desired value.
However the message definition of SetMode seems not present in the repository files 8.hmu.csv or15.ctlv2.csv. 
( https://github.com/john30/ebusd-configuration/tree/master/ebusd-2.1.x )
Can you also provide the message definition for SetMode, so that I can include it in my csv file?
Thanks in advance

**Reply by ralfnap2** [reply, marked as answer]
Hi avanmourik,
the line is not in 8.hmu, it is in include file hcmode.inc

# HC Betriebsart2,,,,,,,,,,,,,
*r,,,,,,"B511",,,,,,,
*uw,,,,,,"B510",,,,,,,
uw,,SetMode,Betriebsart,,,,00,,,hcmode,,,,flowtempdesired,,temp1,,,,hwctempdesired,,temp1,,,,hwcflowtempdesired,,temp0,,,,,,IGN:1,,,,disablehc,,BI0,,,,disablehwctapping,,BI1,,,,disablehwcload,,BI2,,,,,,IGN:1,,,,remoteControlHcPump,,BI0,,,,releaseBackup,,BI1,,,,releaseCooling,,BI2,,,,
#potential last byte: currentmode,s,UCH,0=off;1=standby;2=heat;3=water,,
r,,Status01,Vorlauftemperatur/Rücklauftemperatur/Aussentemperatur/WW Temperatur/Speichertemperatur/Pumpenstatus,,,,01,,,temp1;temp1;temp2;temp1;temp1;pumpstate,,,
r,,Status02,Betriebsart/Maximaltemperatur/ReglerCurrentTEMP/Maximaltemperatur/ReglerCurrentTemp,,,,02,,,hwcmode;temp0;temp1;temp0;temp1,,,
r,,Status,Status,,,,03,,,temp;press;press;hcmode2;HEX,,,

**Reply by avanmourik** [reply]
Hello ralfnap2,
Thanks for your help. 
I had included the hcmode file, but discovered it was not loading due to errors.
After having solved most of the errors, I now get the desired values
hmu SetMode = auto;52.5;-;-;0;1;1;0;0;0
hmu Status01 = 42.5;37.5;-;-;-;on
Will monitor this for a while and then close the issue

**Reply by avanmourik** [reply]
helloralfnap2,
It worked fine with the directions you gave me
